Payments/UserInterface/01a_ListAuthorizations.md

```diff
@@ -0,0 +1,299 @@
+    - **4 - Failure**   
+        The payment service provider has received the request, but it cannot process it. If available for the connection, the status information field may provide further information.
+    - **5 - Error**   
+       Errors have been occurred during transfer.
+    - **6 - Void**   
+       The transaction has been voided, see [Void an authorization transaction](01_ManageAuthorizations.md#void-an-authorization-transaction).
+
+- *Connection*   
+   Name of the Connection. Note that this is the customer-specific name that has been given the connection. For detailed information, see [Establish connections to payment service providers (PSP)](../Integration/01_EstablishConnection.md).
+
+- *Created by*   
+   User name of the person or API who has created the transaction. Can be a real user or an API user.
+
+- *Modified by*   
+   User name of the person or API who has changed the transaction. Can be a real user or an API user.
+
+- *ID*   
+   Actino unique identifier of the transaction.   
+     
+- *Type*   
+   Kind of the displayed transaction, in this case an authorization transaction.  
+
+- *Connection ID*   
+   Connection identification number. The ID number is automatically assigned by the system.    
+
+- *Created At*   
+   Date and time, on which the payment service provider has created the transaction. 
+
+- *Modified At*   
+   Date and time, on which the payment service provider has modified the transaction.  
+
+- *Currency*   
+   Currency in which the payment will be posted.  
+
+- *Amount*   
+   Invoice amount of the payment in the currency in which the payment will be posted.   
+
+- *Reference currency*   
+  Currency of the original payment. 
+
+- *Reference amount*   
+   Currency of the original payment. Makes it possible that the accounting can post the payment to the correct currency account so that a refund later do not lead to currency differences.
+
+- *Name*   
+   First name of the person who has made the payment.   
+
+- *Surname*   
+   Last name of the person who has made the payment.   
+
+- *Company*   
+   Name of the company on whose behalf the payment was made.   
+
+- *Address*   
+   Street and house number of the address.<!---Straße und Hausnummer? oder konkatenierte Adresse? -->     
+
+-  *Postal code*   
+   Zip code of the address.    
+
+- *City*   
+   City of the address.    
+
+- *State*   
+   State of the address.    
+
+- *Country*   
+   Country of the address.   
+
+- *Charges amount*   
+   Amount of the fee.    
+
+- *Charges currency*   
+   Currency of the fee.      
+
+- *Charges other*   
+   Other fees.     
+
+- *Reference*   
+   Mostly, depending on the payment service provider, the order number.
+
+- *Intented use*   
+   In general, depending on the payment service provider, a further description of the transaction.
+
+- *Charges intended use*   
+   Description of the fee, for example transaction fee.
+
+- *Booking date*   
```

Comment:
Booking ist sicherlich die Übersetzung für Buchung. In Buchhaltung haben wir alle Buchungen/Buchen usw. zu Posting/Post... Ich würde es hier auch ändern, auch wenn im UI noch "falsch" steht. Auch bitte die Terminologie Excel dementsprechend aktualisieren.